packages/invariant/index.ts

```diff
@@ -0,0 +1,29 @@
+const isProduction: boolean = process.env.NODE_ENV === "production";
+const prefix = "Visit https://community.frontity.org for help! 🙂\n\n";
+
+// Throw an error if the condition fails
+export function invariant(condition: boolean, message?: string) {
+  if (condition) {
+    return;
+  }
+  // Condition not passed
+
+  if (isProduction) {
+    throw new Error(prefix);
+  } else {
+    throw new Error(`${prefix}${message || ""}`);
+  }
+}
+
+// Use a closure to ensure that the warning only shows up once!
+export const warning = (() => {
+  if (isProduction) {
+    return;
+  }
+  let warned = false;
+
+  return (message?: string) => {
+    if (!warned) console.warn(`${prefix}${message || ""}`);
+    warned = true;
+  };
```

Comment:
I think this closure is wrong because it will trigger the warning only once even for different warnings.

Add tests for more than one warning with the same message and more than one warning with different messages.

Reply:
I think you're right @luisherranz. Not sure how I missed that.

Reply:
fixed.